.mailmap

```diff
@@ -7,8 +7,10 @@ Evan Priestley <epriestley@javelinjs.com> <git@epriestley.com>
 Josh Vera <josh@joshvera.com> <vera@github.com>
 Lars Schneider <larsxschneider@gmail.com> <lars.schneider@autodesk.com>
 Lee Reilly <lee@leereilly.net> <lee@github.com>
+Noam Y. Tenne <noam@10ne.org> noamt <noam@10ne.org>
 Rick Olson <technoweenie@gmail.com> rick <technoweenie@gmail.com>
 Rick Olson <technoweenie@gmail.com> risk danger olson <technoweenie@gmail.com>
+Rick Olson <technoweenie@gmail.com> Your Name <you@example.com>
```

Comment:
No, *from* `Your Name <you@example.com>` to `Rick Olson <technoweenie@gmail.com>`. I remember also stumbling upon a commit from this user when I created my `.mailmap`, but I did not find sufficient proof that this is in fact @technoweenie. I guess @dpursehouse still remembered that this is the case.